src/test/unit/specific_feature_tests/skins.js

```diff
@@ -0,0 +1,36 @@
+/* jshint maxlen: 100 */
+/* global -$,
+    QUnit,
+    equal,
+    module,
+    notEqual,
+    test,
+    prepareUnitTestModule
+*/
+"use strict";
+
+module(
+    "skin-ui-overrides",
+    {
+        setup: function () {
+            prepareUnitTestModule({
+                options: {skin: 'basic'}
```

Comment:
According to [this line in the test below](https://github.com/wymeditor/wymeditor/pull/710/files#diff-6567671bc27f9cb5c72677b1981b278aR35), it seems that `'basic'` is not the default.

I'm confused, because [changes like this](https://github.com/wymeditor/wymeditor/pull/710/files#diff-209fc1960195c91ceff0368bcbe4b835L9) make it seem that we are changing the default to be `'basic'`.